content/en/docs/external-add-ons/feast/introduction.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Introduction"
+description = "A brief introduction to Feast and Feature Stores"
+weight = 10
++++
+
+## What is Feast?
+
+[Feast](https://docs.feast.dev/) is an [open-source](https://github.com/feast-dev/feast) feature store that helps teams operate ML systems at scale by allowing them to define, manage, validate, and serve features to models in production. 
+
+Feast provides the following functionality:
+
+- __Load streaming and batch data__: Feast is built to be able to ingest data from a variety of bounded or unbounded sources. 
```

Comment:
```suggestion
- __Load streaming, batch, and request-time data__: Feast is built to be able to ingest data from a variety of bounded or unbounded sources. 
```

Reply:
Done in https://github.com/kubeflow/website/pull/3836/commits/a5e72b551cea20a364a90aed4f6ef4f558a2582e